tests/test_h5store.py

```diff
@@ -661,13 +664,16 @@ def test_single_writer_multiple_reader_same_process(self):
                         self.assertEqual(reader1['test'], True)
                         self.assertEqual(reader2['test'], True)
 
+    @unittest.skipIf(WINDOWS, 'This test fails for an unknown reason on Windows.')
     def test_single_writer_multiple_reader_same_instance(self):
         from multiprocessing import Process
 
         def read():
             p = Process(target=_read_from_h5store, args=(self._fn_store,), kwargs=(dict(mode=None)))
```

Comment:
I am sure it already defaults to this, but since we are giving `kwargs` anyways `kwargs=(dict(mode='r'))` would be more explicit.

Reply:
Good call. Fixed.